### PR TITLE
Address remaining dependabot alerts

### PR DESCRIPTION
**Pull request by cocreature**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: compiler/daml-extension/yarn.lock [hunk 1]
```diff
@@ -8,17 +8,17 @@
   integrity sha512-RSu6I5yVVtjDP6PTkhDVvqOC+KJf8t4Uo2qCLxGjunLVt4guqKLt3KQX+dEU/qVn5vECXGTprDSNk+c9G6hDmg==
 
 "@types/node-fetch@^2.5.3":
-  version "2.5.10"
-  resolved "https://registry.yarnpkg.com/@types/node-fetch/-/node-fetch-2.5.10.tgz#9b4d4a0425562f9fcea70b12cb3fcdd946ca8132"
-  integrity sha512-IpkX0AasN44hgEad0gEF/V6EgR5n69VEqPEgnmoM8GsIGro3PowbWs4tR6IhxUTyPLpOn+fiGG6nrQhcmoCuIQ==
+  version "2.5.12"
```
Comment (cocreature) [inline compiler/daml-extension/yarn.lock:11]: I got tired of doing selective updates and just regenerated the stupid yarn.lock. Verified manually that it still works.

Comment (cocreature) [reply compiler/daml-extension/yarn.lock]: except ofc this doesn’t compile on Windows so reverted

**Review by stefanobaghino-da — APPROVED** [review]
Luckily we have `dependabot` taking care of security. ❤️ 🤖

**Comment by cocreature** [comment]
> Luckily we have `dependabot` ~taking care of security~ wasting Moritz’ time
ftfy

**Pull request merged**